src/semidiscretization/semidiscretization_coupled.jl

```diff
@@ -123,6 +123,10 @@ end
     sum(ndofs, semi.semis)
 end
 
+@inline function ndofsglobal(semi::SemidiscretizationCoupled)
+    sum(ndofsglobal, semi.semis)
+end
```

Comment:
```suggestion
"""
    ndofsglobal(semi::SemidiscretizationCoupled)
    
Return the global number of degrees of freedom associated with each scalar variable across all MPI ranks, and summed up over all coupled systems.
This is the same as [`ndofs`](@ref) for simulations running in serial or
parallelized via threads. It will in general be different for simulations
running in parallel with MPI.
"""
@inline function ndofsglobal(semi::SemidiscretizationCoupled)
    sum(ndofsglobal, semi.semis)
end
```

Reply:
I would actually argue that this docstring doesn't really help in practice since it's the same as the one for the `AbstractSemidiscretization` - and `SemidiscretizationCoupled <: AbstractSemidiscretization`. But I don't have a strong opinion on this. Shall we keep it, @sloede?

Reply:
Well, it explicitly mentions that it is the number of DOFs over all coupled systems. I don't have a strong opinion either, so I'll leave it up to @benegee to decide 🙂